code/def_files/data/effects/main-f.sdr

```diff
@@ -293,10 +293,17 @@ void main()
  #ifdef FLAG_TEAMCOLOR
 	vec4 teamMask = vec4(0.0, 0.0, 0.0, 0.0);
 	teamMask = texture(sMiscmap, vec3(texCoord, float(sMiscmapIndex)));
-	vec3 base = max(base_color - vec3(0.5), vec3(0.0));
-	vec3 stripe = max(stripe_color - vec3(0.5), vec3(0.0));
-	baseColor.rgb += (base * teamMask.x) + (stripe * teamMask.y);
+	vec3 team_color = (base_color * teamMask.x) + (stripe_color * teamMask.y);
+	vec3 team_color_glow = (base_color * teamMask.b) + (stripe_color * teamMask.a);
+	#ifdef FLAG_HDR
+	team_color = pow(team_color, vec3(SRGB_GAMMA));
+	team_color_glow = pow(team_color_glow, vec3(SRGB_GAMMA)) * 3.0f;
+	#endif
+
+	baseColor.rgb += team_color;
 	baseColor.rgb = max(baseColor.rgb, vec3(0.0));
+	specColor.rgb += team_color;
+	specColor.rgb = max(specColor.rgb, vec3(0.03f));
```

Comment:
There are a few instances in the other changes but GLSL does not require the `f` for specifying a `float` value.